configs/components/cpp-pcp-client.rb

```diff
@@ -11,7 +11,12 @@
     pkg.environment "PATH", "#{settings[:bindir]}:/opt/pl-build-tools/bin:$(PATH)"
   end
 
-  pkg.build_requires "openssl"
+  if !settings[:vendor_openssl]
+    pkg.build_requires 'openssl-devel'
```

Comment:
I have adjusted the commit message to include the reason for the change. 

Yes I have been aware of the reasons for vendor-ing openssl and we are not intending to undo/remove that broadly. This is only for generating fips compliant builds for certain targets - would be very conscious.